docs/e2eTestPrompts.md

```diff
@@ -154,6 +154,14 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | azmcp_eventgrid_subscription_list | Show Event Grid subscriptions in resource group <resource_group_name> in subscription <subscription> |
 | azmcp_eventgrid_subscription_list | List Event Grid subscriptions for subscription <subscription> in location <location> |
 
+## Azure Event Hubs
+
+| Tool Name | Test Prompt |
+|:----------|:----------|
+| azmcp_eventhubs_namespace_get | List all Event Hubs namespaces in my subscription |
+| azmcp_eventhubs_namespace_get | Get the details of my Event Hubs namespace <namespace_id> |
```

Comment:
nit: consistent change `<namespace_id>` to `<namespace_name>` to match one below

Reply:
@conniey <namespace_name> is whatever you named the namespace within your resource group while <namespace_id> is the complete identifier for the resource. The backing ARM libraries supports namespace discovery via the id and the resource name and resource group combination.